cypress/integration/comparator.spec.ts

```diff
@@ -0,0 +1,2 @@
+// TODO: implement e2e for comparator here
+export {}
```

Comment:
This empty export is necessary due to `isolatedModules` TS option.

Reply:
I have no clue what that means, so I'll just trust you on this. 😄

Reply:
You have a detailed explanation here: https://www.typescriptlang.org/tsconfig#non-module-files